core/consensus/pow/primitives/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use substrate_client::decl_runtime_apis;
+use rstd::vec::Vec;
+use sr_primitives::ConsensusEngineId;
+use primitives::H256;
+
+/// The `ConsensusEngineId` of PoW.
+pub const POW_ENGINE_ID: ConsensusEngineId = [b'p', b'o', b'w', b'_'];
```

Comment:
PoW is a class of algorithms. Consider rename to the actual PoW algorithm name.

Reply:
The current implementation is designed to be generic. We only define the absolute minimal things needed to carry out PoW fork choice in consensus engine. Using this, the runtime should be able to implement any class of PoW algorithms.

In other words, there's nothing we can rename to other than `GenericPow`. :)